src/Components/ArtworkAlert/Components/Form/ArtworkAlertNameInput.tsx

```diff
@@ -0,0 +1,96 @@
+import { FC, useEffect, useState } from "react"
+import { Input } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useFormikContext } from "formik"
+
+import { ArtworkAlertFormikValues } from "Components/ArtworkAlert/Components/Steps/Details"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { useArtworkAlertContext } from "Components/ArtworkAlert/Hooks/useArtworkAlertContext"
+import { useDebouncedValue } from "Utils/Hooks/useDebounce"
+import { useSystemContext } from "System/SystemContext"
+
+import { ArtworkAlertNameInputQuery } from "__generated__/ArtworkAlertNameInputQuery.graphql"
+import { ArtworkAlertNameInput_previewSavedSearch$data } from "__generated__/ArtworkAlertNameInput_previewSavedSearch.graphql"
+
+export const ArtworkAlertNameInputQueryRenderer: FC = () => {
+  const { relayEnvironment } = useSystemContext()
+
+  const { state } = useArtworkAlertContext()
```

Comment:
**walkthrough**: This file is almost identical to the one used in the existing flow with the exception of using the ArtworkAlertContext instead.